lianad/src/database/sqlite/schema.rs

```diff
@@ -122,6 +122,28 @@ CREATE TABLE labels (
     item TEXT UNIQUE NOT NULL,
     value TEXT NOT NULL
 );
+
+/* Payjoin OHttpKeys */
+CREATE TABLE payjoin_ohttp_keys (
+    id INTEGER PRIMARY KEY NOT NULL,
+    relay TEXT UNIQUE NOT NULL,
+    timestamp INTEGER NOT NULL,
+    keys BLOB NOT NULL
```

Comment:
Is it many keys or just the one key

Reply:
That one was indeed tricky to name...it is one `OhttpKeys` 🤔 
But I think using `key` would make more sense here.